server/docs/hellobooks_api_doc.yaml

```diff
@@ -543,6 +543,55 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/errorResponse"
+  /books/borrow:
+    post:
+      tags:
+        - Books
+      security:
+        - bearerAuth: []
+      summary: lend a book
+      description: Admin should be able to lend out books
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                isbn:
+                  type: integer
+                title:
+                  type: string
+                patronId:
+                  type: integer
+                cost:
```

Comment:
Where is this used?

Reply:
I'll remove it, it's not meant to be there. I initially had it in the model but I later removed it because we've concluded that payments will be based on monthly subscription.